charts/rstudio-library/NEWS.md

```diff
@@ -1,3 +1,21 @@
+# 0.1.22
+
+- Add support for list of INI file sections, like the following example
+
+    ```yaml
+    config:
+      server:
+        launcher.conf:
+          cluster:
+            - name: Cluster1
+              type: Kubernetes
+            - name: Cluster2
+              type: Kubernetes
+              config-file: /path/to/config/file
+    ```
+
+- Fixes issue where RBAC service account is always created even when `serviceAccount.create: false`
```

Comment:
Maybe make a note here that this now requires `serviceAccount.create` to be provided as a boolean (and not empty)?